sdk/formrecognizer/azure-ai-formrecognizer/samples/README.md

```diff
@@ -16,7 +16,7 @@ urlFragment: formrecognizer-samples
 > code from client library version 3.1.X or lower to the latest version. Additionally, see the [Changelog][changelog] for more detailed information.
 
 These code samples show common scenario operations with the Azure Form Recognizer client library.
-The async versions of the samples require Python 3.6 or later.
+The async versions of the samples require Python 3.7 or later.
```

Comment:
we can just remove this sentence since all versions supported have async support